### DISCUSSION TITLE
Add `@ganerated` tag to automatically-generated Rust files

### DISCUSSION THREAD

**Discussion by Jlobblet** [diesel-rs/diesel#3105] [Ideas]
[`rustfmt` can be configured to format (or not) files that contain `@generated` in the first 5 lines.](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#format_generated_files) Diesel should emit this tag in files it automatically generates such as `schema.rs`.

**Comment by weiznich** [comment]
That's already implemented in https://github.com/diesel-rs/diesel/pull/2462. Its not part of any release yet.

**Reply by Jlobblet** [reply]
Ah, my error. I thought I had searched all PRs and issues but evidently not. Thanks!
